model/countries/MX/MX-comparison-stopwords.yaml

```diff
@@ -0,0 +1,3 @@
+in-building-location: Piso
+landmark-and-cross-streets: Entre Calles
```

Comment:
If there is a whitespace in the stopword, does it require `"`? Otherwise we can drop the`"` from  "Ponto de referência:" above.

Reply:
As discussed in person. We don't need quotes here because yaml interprets this as a string. We need them for `Ponto de referência:` because of the colon that gets interpreted by yaml otherwise.